### DISCUSSION TITLE
Need help configuring OpenFGA server with MySQL hosted on cloud.

### DISCUSSION THREAD

**Discussion by S4nfs** [openfga/community#273] [Q&A]
I am configuring openFGA standalone server without docker & k8s but i am getting an error: 
`invalid database uri: default addr for network 'blueeit-sagar2nfs-7dbe.a.aivencloud.com:22395' unknown` 

and i am passing parameters like 
`./openfga migrate --datastore-engine mysql --datastore-uri 'mysql://avnadmin:mypassword@blueeit-sagar2nfs-7dbe.a.aive
ncloud.com:22395/openfga?parseTime=true&ssl-mode=REQUIRED'`

Is remote uri supported in openFGA? Also i am not seeing any config.yaml as described in the docs while setting up with different method.

One last question is that can i migrate tuples from one model to another? , e.g if i created a different authorization model which adds some other relationships or features on top of my older model but need to keep the existing tuples/touple queries. I found this here: [https://openfga.dev/docs/modeling/migrating/migrating-relations](https://openfga.dev/docs/modeling/migrating/migrating-relations) but it seems i have to manually recreate all relationship tuples again for the new model.

**Comment by aaguiarz** [comment]
Hi @S4nfs 

About migration, you don't need to migrate the data if you are adding new types and relations. You update the model, and the existing tuples will still be available and OpenFGA will use them.  

If you are creating a new store and want to copy the data, you can read all the tuples with the CLI using `fga tuple read` and write it with `fga tuple write`, e.g
```
fga tuple read  --store-id STORE1 --output-format=simple-json --max-pages=0 > tuples.json
fga tuple write --store-id STORE2 --file tuples.json
```
